### PR TITLE
🎨 Make customization a bit easier again

### PR DESCRIPTION
**Pull request by Pocket-titan**

### CODE REVIEW

**Comment by fonsp** [comment]
Nice, I was also hoping to have a `light.css` that defines the current theme colors using CSS variables, and then `editor.css` can use those variables. Changing the theme would be swapping out `light.css` for `rainbow.css`. That way, `editor.css` can stay the same for all themes (and we will still only have one style sheet to maintain).

File: frontend/components/CellInput.js [hunk 1]
```diff
@@ -1,7 +1,19 @@
 import { html, useState, useEffect, useRef } from "../common/Preact.js"
-
 import { utf8index_to_ut16index } from "../common/UnicodeTools.js"
 
+// For customization, see: https://codemirror.net/doc/manual.html#config
+// Note that some options will require you to add additional addons in editor.html
+const custom_codemirror_options = {
```
Comment (fonsp) [inline frontend/components/CellInput.js:6]: Why does this make customization easier?

Comment (fonsp) [reply frontend/components/CellInput.js:6]: Maybe add a comment explaining how people can customize these? (A fork?)

**Comment by Pocket-titan** [comment]
> Nice, I was also hoping to have a `light.css` that defines the current theme colors using CSS variables, and then `editor.css` can use those variables. Changing the theme would be swapping out `light.css` for `rainbow.css`. That way, `editor.css` can stay the same for all themes (and we will still only have one style sheet to maintain).

That sounds good, but by "theme" do you mean the Pluto styles excluding synax highlighting/the codemirror editor, or including? Right now I have them separated (they could be separated more still) to match how codemirror handles themes

File: frontend/components/CellInput.js (hunk 1 above)
Comment (Pocket-titan) [reply frontend/components/CellInput.js:6]: Right now it's just a separate object containing settings the user should be able to change (e.g. don't change `mode: "julia"`), but I'm not sure yet about the implementation (forking isn't ideal b/c every update will give merge conflicts), maybe a settings menu that persists to localstorage?

**Comment by fonsp** [comment]
Both, they should have matching color schemes

**Pull request closed without merging**